src/analysis/retail/hunter/marksmanship/modules/talents/AimedShot.tsx

```diff
@@ -77,9 +88,23 @@ class AimedShot extends Analyzer {
      * Dead Eye and Trueshot scale multiplicatively off each other, which can lead to extremely fast cooldown reduction that this should properly handle.
      */
     let modRate = 1;
+
+    // Sum the total cdr reduction from Trueshot, Tactial Reload, Steady Focus
+    let reduction = 0;
     if (this.selectedCombatant.hasBuff(SPELLS.TRUESHOT.id)) {
-      modRate /= 1 + TRUESHOT_AIMED_SHOT_RECHARGE_INCREASE;
+      reduction += TRUESHOT_AIMED_SHOT_RECHARGE_INCREASE;
+    }
+
+    if (this.selectedCombatant.hasTalent(TALENTS_HUNTER.TACTICAL_RELOAD_TALENT)) {
```

Comment:
Since this talent statically reduces the cooldown - you can just do handle it in the abilities file. You should be able to find many examples of spells being reduced with X talent in there.

Reply:
I think I addressed it? Not entirely sure, feel free to take a look and lmk if thats what you meant.

I moved the Tactical reload talent cdr logic to aimed shot in abilities file.